app/models/devise_token_auth/concerns/user.rb

```diff
@@ -254,4 +255,8 @@ def clean_old_tokens
       tokens.shift while max_client_tokens_exceeded?
     end
   end
+
+  def token_lifespan
+    DeviseTokenAuth.token_lifespan
```

Comment:
now, what's the purpose of this code https://github.com/lynndylanhurley/devise_token_auth/blob/ce2cf2c180e6632dd054dd4b099e27f5a264de83/lib/devise_token_auth/token_factory.rb#L63 ?